wgsl/index.bs

```diff
@@ -491,6 +498,18 @@ An <dfn>integer literal</dfn> is:
     * `0x` or `0X` followed by a sequence of hexadecimal digits.
 * Then an optional `i` or `u` suffix.
 
+<div class='example wgsl int-literals' heading='integer literals'>
+  <xmp highlight='rust'>
+    const a = 0x123;
+    const b = 0X123u;
+    const c = 1u;
+    const d = 123;
+    const f = 0;
+    const f = 0i;
```

Comment:
Let's not repeat 'f'.  'e' is missing, though.

Reply:
done